src/ViewLayer/Faults/MotorFaults/MotorFaultList.cpp

```diff
@@ -34,143 +34,84 @@ MotorFaultList::MotorFaultList()
-}
-
-QVector<FaultLabel>& MotorFaultList::limitLabels()
-{
-    return limitLabels_;
+    return faultLabels_;
 }
 
 void MotorFaultList::updateErrors(const ErrorFlags& errorFlags)
 {
-    errorLabels_[0].setActive(errorFlags.motorOverSpeed());
-    errorLabels_[1].setActive(errorFlags.softwareOverCurrent());
-    errorLabels_[2].setActive(errorFlags.dcBusOverVoltage());
-    errorLabels_[3].setActive(errorFlags.desaturationFault());
-    errorLabels_[4].setActive(errorFlags.badMotorPositionHallSequence());
-    errorLabels_[5].setActive(errorFlags.watchdogCausedLastReset());
-    errorLabels_[6].setActive(errorFlags.configReadError());
-    errorLabels_[7].setActive(errorFlags.railUnderVoltageLockOut());
+    faultLabels_[0].setActive(errorFlags.motorOverSpeed());
+    faultLabels_[1].setActive(errorFlags.softwareOverCurrent());
+    faultLabels_[2].setActive(errorFlags.dcBusOverVoltage());
+    faultLabels_[3].setActive(errorFlags.desaturationFault());
+    faultLabels_[4].setActive(errorFlags.badMotorPositionHallSequence());
+    faultLabels_[5].setActive(errorFlags.watchdogCausedLastReset());
+    faultLabels_[6].setActive(errorFlags.configReadError());
+    faultLabels_[7].setActive(errorFlags.railUnderVoltageLockOut());
 }
 
 void MotorFaultList::updateLimits(const LimitFlags& limitFlags)
 {
-    limitLabels_[0].setActive(limitFlags.outputVoltagePwmLimit());
-    limitLabels_[1].setActive(limitFlags.motorCurrentLimit());
-    limitLabels_[2].setActive(limitFlags.busCurrentLimit());
-    limitLabels_[3].setActive(limitFlags.busVoltageUpperLimit());
-    limitLabels_[4].setActive(limitFlags.busVoltageLowerLimit());
-    limitLabels_[5].setActive(limitFlags.ipmOrMotorTemperatureLimit());
-    limitLabels_[6].setActive(limitFlags.velocityLimit());
+    faultLabels_[0 + NUMBER_OF_MOTOR_ERRORS].setActive(limitFlags.outputVoltagePwmLimit());
+    faultLabels_[1 + NUMBER_OF_MOTOR_ERRORS].setActive(limitFlags.motorCurrentLimit());
+    faultLabels_[2 + NUMBER_OF_MOTOR_ERRORS].setActive(limitFlags.busCurrentLimit());
+    faultLabels_[3 + NUMBER_OF_MOTOR_ERRORS].setActive(limitFlags.busVoltageUpperLimit());
+    faultLabels_[4 + NUMBER_OF_MOTOR_ERRORS].setActive(limitFlags.busVoltageLowerLimit());
+    faultLabels_[5 + NUMBER_OF_MOTOR_ERRORS].setActive(limitFlags.ipmOrMotorTemperatureLimit());
+    faultLabels_[6 + NUMBER_OF_MOTOR_ERRORS].setActive(limitFlags.velocityLimit());
 }
 
 int MotorFaultList::numberOfActiveLabels() const
 {
     int numberOfLabels = 0;
 
-    for (int i = 0; i < errorLabels_.size(); i++)
+    for (int i = 0; i < faultLabels_.size(); i++)
     {
-        if (errorLabels_[i].isActive())
-        {
-            numberOfLabels++;
-        }
-    }
-
-    for (int i = 0; i < limitLabels_.size(); i++)
-    {
-        if (limitLabels_[i].isActive())
+        if (faultLabels_[i].isActive())
         {
             numberOfLabels++;
         }
     }
 
     return numberOfLabels;
 }
+
+int MotorFaultList::nextFaultIndex()
```

Comment:
imo, I don't think this does enough to warrant it's own function:

1. Technically, this returns the current fault, then increments the value, so the `nextFaultIndex()` name isn't entirely accurate.
2. You can accomplish the same thing in the for loop without really affecting readability
3. No one else is using this function